### ISSUE TITLE
v3 fails to handle equal sign in Unicode.

### ISSUE THREAD

**Issue by t-k-** [mathjax/MathJax#2309]
MathJax [v3 demo page](https://mathjax.github.io/MathJax-demos-web/input-tex2chtml.html) has issue rendering this formula containing an Unicode equal sign:
```
$\sum_{k＝1}$
```
The error message is
> Cannot read property 'parentNode' of null

I am not sure if the demo page is using some outdated version or does this issue still exist in latest version.

**Comment by dpvc** [comment]
The issue stems from the fact that your equal sign is form the full-width character range, and the MathJax fonts don't include that character.  That means MathJax needs to try to measure the character in the browser, since it has no metric information for it.  The demonstration you link to is an illustration of the `MathJax.tex2chtml()` function, which calls the `convert()` function internally.  That converts a TeX string to DOM elements without the need for the string to be in the document itself.  But when MathJax needs to measure unknown characters, it must insert it into the document to do so.  MathJax inserts the character in the document at the location of the original math string, but in the case of `convert()`, there is no such location, and that is what is leading to this error.

I have made a pull request that allows `convert()` to handle unknown characters without throwing an error.
